packages-backend/loggers/src/loggers.spec.ts

```diff
@@ -2,47 +2,73 @@ import createApplicationLogger from './create-application-logger';
 import rewriteFieldsFormatter from './formatters/rewrite-fields';
 
 describe('application logger', () => {
-  it('should log and rewrite fields', () => {
-    // @ts-ignore
-    console._stdout.write = jest.fn();
-    const logger = createApplicationLogger({
-      json: true,
-      formatters: [
-        rewriteFieldsFormatter({
-          fields: [
-            {
-              from: 'meta.req.headers',
-              to: 'meta.req.headersJsonString',
-              replaceValue: (value) => {
-                return JSON.stringify(value);
+  describe('when replacing field', () => {
+    it('should log and replace fields', () => {
+      // @ts-ignore
+      console._stdout.write = jest.fn();
+      const logger = createApplicationLogger({
+        json: true,
+        formatters: [
+          rewriteFieldsFormatter({
+            fields: [
+              {
+                from: 'meta.req.headers',
+                to: 'meta.req.headersJsonString',
+                replaceValue: (value) => {
+                  return JSON.stringify(value);
+                },
               },
-            },
-          ],
-        }),
-      ],
+            ],
+          }),
+        ],
+      });
+      logger.info('Test log', {
+        meta: { req: { headers: { Accept: 'application/json' } } },
+      });
+      // Note: that both only headersJsonString is present
+      expect(console._stdout.write.mock.lastCall).toMatchInlineSnapshot(`
```

Comment:
I suggest to move to inline snapshots. While the string matchers are not hardened to change they're also troublesome to debug. As a tradeoff a added a comment.